packages/desktop-client/src/components/manager/BudgetList.tsx

```diff
@@ -197,7 +206,7 @@ function FileItem({
   return (
     <View
       onClick={() => _onSelect(file)}
-      title={getFileDescription(file) || ''}
+      title={description ? t(description) : ''}
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏could you move `t` to inside `getFileDescription` instead? The translations should be as close to the content as possible.

Reply:
Sure, done!